### PR TITLE
Fix/mongo document

### PR DESCRIPTION
**Pull request by BenxinNiu**

### CODE REVIEW

File: buildspec.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ phases:
   post_build:
     commands:
       - echo Build completed on `date`
-      - version=0.0.2
+      - version=0.0.3
       - artifact_name=sensor-inventory-dist
```
Comment (BenxinNiu) [inline buildspec.yml:21]: increase patch version for release

File: src/app.py [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
+
 app = connexion.App(__name__, specification_dir='./')
 app.add_api('swagger.yaml')
+DB.connect_db()
+
 
-#test to insert data to the data base, meant to be removed later
+# test to insert data to the data base, meant to be removed later
+# TODO: remove this... and implement unit test
 @app.route("/test")
```
Comment (BenxinNiu) [inline src/app.py:12]: update test endpoint

File: src/repositories/db.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
-from flask_pymongo import pymongo
-from .connection_string import MONGO_CONNECTION_STRING
-from mongoengine import *
-import ssl
+from .mongo_config import MONGO_CONNECTION_STRING, DATABASE_NAME, MONGO_CERT
+from mongoengine import connect
```
Comment (BenxinNiu) [inline src/repositories/db.py:2]: please avoid using `*` wild card import as it is not a good practice, in terms of readability and performance

File: src/repositories/models.py [hunk 4]
```diff
@@ -1,17 +1,31 @@
 from mongoengine import *
+from enum import Enum
 
-class Organization(Document):
+
```
Comment (BenxinNiu) [inline src/repositories/models.py:4]: Both `Orgnization` and `Machine` objects should be embedded object under the `Sensor` object

File: src/repositories/models.py [hunk 5]
```diff
@@ -1,17 +1,31 @@
...
-    organization_id = ReferenceField(Organization, required=False)
 
 class Sensor(Document):
+    id = StringField(required=False)
     sensor_name = StringField(required=True)
-    organization_id = ReferenceField(Organization, required=True)
-    machine_id = ReferenceField(Machine, required=True)
+    organization = EmbeddedDocumentField(Organization, required=False)
+    machine = EmbeddedDocumentField(Machine, required=False)
     sensor_install_time = DateField(required=False)
```
Comment (BenxinNiu) [inline src/repositories/models.py:24]: Please be mindful of the Object type

File: src/repositories/models.py [hunk 6]
```diff
@@ -1,17 +1,31 @@
...
-    sensor_status = StringField(required=True, max_length=50)
\ No newline at end of file
+    sensor_status = StringField(required=True, max_length=50)
+
+
+class SensorStatus(str, Enum):
```
Comment (BenxinNiu) [inline src/repositories/models.py:28]: Adding Enum class for sensor status as per defined in swagger file

File: src/repositories/mongo_config.py [hunk 7]
```diff
@@ -1 +1,5 @@
-MONGO_CONNECTION_STRING = "mongodb+srv://db-vape-sensor-inventory:theMostSecurePassword?!@cluster-vape-sensor-inventory-fcfqt.mongodb.net/test?retryWrites=true&w=majority"
\ No newline at end of file
+import ssl
+
```
Comment (BenxinNiu) [inline src/repositories/mongo_config.py:2]: update connection string to use `inventory` database instead of `test` as previous

**Review by Summer0410 — APPROVED** [review]

**Pull request merged**